modules/plot/test/spec.coffee

```diff
@@ -243,15 +254,24 @@ describe "plot", ->
       checkOptionAndSetterGetter(hx.BarSeries, 'group', ['group1', 'group2'])
 
     describe "render", ->
-      it 'should display when the height is positive', ->
```

Comment:
I think this test should probably be left in as it's testing the existing method for creating graphs